sound/soc/sof/intel/hda-dsp.c

```diff
@@ -704,6 +704,9 @@ static int hda_resume(struct snd_sof_dev *sdev, bool runtime_resume)
 	hda_dsp_ctrl_ppcap_enable(sdev, true);
 	hda_dsp_ctrl_ppcap_int_enable(sdev, true);
 
+	/* display codec can powered off after controller init */
+	hda_codec_i915_display_power(sdev, false);
+
```

Comment:
@kv2019i I think we should also remove the display_power ON in hda_dap_resume() when resuming from D0i3 and the display power OFF in hda_dsp_suspend when entering D0i3. Those are not needed anymore as well, isnt it?

Reply:
apparently this is working correctly, but I'm just a bit surprised that the display power isn't managed in a counted `get()` / `put()` way. But well, I guess there are reasons for that.

Reply:
@lyakh fair point. We used to have get/put functions on SOF side, but their implementations were not refcount based, making it even more confusing.  Basicly SOF just signals one bit to the HDAC core, which has its own refcounted logic to track users of the display power. So basicly we really have a single signal we control in SOF, so thus the boolean control we have. This is matching the design in legacy driver, making it a bit easier to keep the two in sync.

Reply:
Ack @ranj063 , let me change that.